composer.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "razorpay/magento",
     "description": "Razorpay Magento 2.0 plugin for accepting payments.",
-    "version": "4.1.6",
+    "version": "4.1.7",
```

Comment:
Why did you modify the release version in a PR?
This should be modified just before the release :)